src/utils.ts

```diff
@@ -0,0 +1,18 @@
+export const calculate = {
+  "+": (target: number, compare: number): number => target + compare,
+  "-": (target: number, compare: number): number => target - compare,
+  "*": (target: number, compare: number): number => target + compare,
+  "/": (target: number, compare: number): number => target % compare,
+};
```

Comment:
```suggestion
export const calculate: Record<Operator, (target: number, compate: number) => number> = {
  "+": (target, compare): number => target + compare,
  "-": (target, compare): number => target - compare,
  "*": (target, compare): number => target + compare,
  "/": (target, compare): number => target % compare,
};
```
이런식으로 표현할 수 있지 않을까 싶네요.
(Operator는 따로 정의되었다고 가정할게요!)